Docs: Add version tags for RC and experimental features (#68840)
vercel/next.js: docs/02-app/02-api-reference/02-file-conventions/instrumentation.mdx

### `onRequestError` (optional)

> > This feature is available from Next.js canary.

Author: ```suggestion
> This API is available in Next.js canary.
```
